srcs/semaphore.cc

```diff
@@ -139,59 +132,32 @@ void Semaphore::Acquire(const Nan::FunctionCallbackInfo<v8::Value>& info)
     return ;
   }
   obj->locked = true;
-  if (obj->debug) {
-    int sval;
-    if (sem_getvalue(obj->semaphore, &sval)) sval = -1;
-    printf("[posix-semaphore] After sem_wait, value = %d\n", sval);
-  }
 }
 
 void Semaphore::Release(const Nan::FunctionCallbackInfo<v8::Value>& info)
 {
   Semaphore* obj = ObjectWrap::Unwrap<Semaphore>(info.Holder());
-  
-  /*if (!obj->strict && (obj->closed || !obj->locked))
```

Comment:
Is there a reason to remove this block? Maybe it was left out after you were done debugging?

Reply:
Ah sorry, failed to see that it was commented out.